packages/sanes-chrome-extension/src/extension/background/model/persona/config/configurationfile.ts

```diff
@@ -50,3 +52,17 @@ const loadConfigurationFile = async (): Promise<ConfigurationFile> => {
 };
 
 export const getConfigurationFile = singleton<typeof loadConfigurationFile>(loadConfigurationFile);
+
+/**
+ * Gets a chain name from the configuration file. Falls back to the chain ID
+ * if no name is found.
+ */
+export async function getChainName(chainId: ChainId): Promise<string> {
+  const chainNames = (await getConfigurationFile()).names;
+
+  if (!Object.keys(chainNames).includes(chainId)) {
```

Comment:
Why simply not `chainNames.hasOwnProperty(chainId);` or `chainId in chainNames`?

Reply:
You are right, `hasOwnProperty` is more direct and does the job. `in` also returns true for all kind of names in the prototype chain, e.g.

```
> "toString" in {}
true
```

Reply:
Yes, `in` will search in prototype chain.